src/test/java/com/synopsys/integration/alert/performance/model/SlackPerformanceProperties.java

```diff
@@ -0,0 +1,37 @@
+package com.synopsys.integration.alert.performance.model;
+
+import com.synopsys.integration.alert.channel.slack.SlackChannelKey;
+import com.synopsys.integration.alert.util.TestProperties;
+import com.synopsys.integration.alert.util.TestPropertyKey;
+
+public class SlackPerformanceProperties extends TestProperties {
```

Comment:
Another instance where I'm not convinced inheritance is the right approach. Is this class adding new functionality to `TestProperties` or is it wrapping functionality of `TestProperties`?